install-and-configure/advanced-configuration/network-costs-configuration.md

```diff
@@ -57,7 +57,14 @@ You can view a list of common config options in this [_values.yaml_](https://git
     extraArgs:
       - "-v=0"
   ```
+Note: for kubecost-network-costs:v0.17.0+ LOG_LEVEL is the env var for setting log level. Valid inputs are error, warn, info, debug, trace.
```

Comment:
Off the top of your head do you know what release we updated the kubecost-network-costs image to 0.17.0?

Reply:
Looks like it was 1.107. With versioning being implemented, we should just remove this stipulation and assume it as the default